client/src/pages/pot-instance/components/LandingPage/styles.module.css

```diff
@@ -85,3 +84,38 @@
 #startHotPotBtn {
   margin-top: 25px;
 }
+
+@media (max-width: 900px) {
```

Comment:
Are these standardized sizes?

Reply:
These are MUI defined sizes. https://mui.com/material-ui/customization/breakpoints/.

To use the mediabreakpoints via mui is kind of annoying though. 
![image](https://user-images.githubusercontent.com/107276941/205459752-49a96fae-96cd-4ea5-837f-ec452fd7aa7b.png)

We needed to have used it when defining all our styles in the first place which is kind of a pain to do now.

Reply:
Ah gotcha. Yeah its better to do all in one epic in the future then.